packages/utilities/psammead-locales/moment/so.js

```diff
@@ -0,0 +1,27 @@
+var moment = require('moment');
+
+var months = [
```

Comment:
Can we keep this consistent. By having `months.split('_')`

Reply:
Happy to use months.split('_') for consistency, but I think is more readable in this way.

Reply:
I agree, but we should ensure consistency across all `moment` overrides

Reply:
Done :)

Reply:
Done :)